src/Tribe/RSVP.php

```diff
@@ -705,11 +713,18 @@ public function send_tickets_email( $order_id ) {
 		}
 
 		$content     = apply_filters( 'tribe_rsvp_email_content', $this->generate_tickets_email_content( $to_send ) );
-		$headers     = apply_filters( 'tribe_rsvp_email_headers', array( 'Content-type: text/html' ) );
-		$attachments = apply_filters( 'tribe_rsvp_email_attachments', array() );
-		$to          = apply_filters( 'tribe_rsvp_email_recipient', $to );
+
```

Comment:
Yeah! I was in doubt about doing it or not, but def seems like the way to go. Added.